src/expr/scalar/func.rs

```diff
@@ -639,6 +639,26 @@ fn round_decimal<'a>(a: Datum<'a>, b: Datum<'a>, a_scale: u8) -> Datum<'a> {
     Datum::from(decimal.round(round_to).significand())
 }
 
+fn convert_from<'a>(a: Datum<'a>, b: Datum<'a>) -> Result<Datum<'a>, EvalError> {
+    let encoding_name = b.unwrap_str().to_lowercase();
+    if encoding_name != "utf8" {
+        return Err(EvalError::InvalidEncodingName(encoding_name));
+    }
+
+    let bytes = a.unwrap_bytes();
+    match from_utf8(bytes) {
+        Ok(from) => Ok(Datum::String(from)),
+        Err(_e) => {
+            let mut byte_sequence = String::new();
+            strconv::format_bytes(&mut byte_sequence, &bytes);
+            Err(EvalError::InvalidByteSequence {
+                byte_sequence,
```

Comment:
Rather than encoding the byte sequence as a hex string, this should do the same thing as `length` and just stuff `e.to_string()` into the `byte_sequence` field, as that will contain details about which specific chunk of the string contained invalid bytes.